tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -79,7 +80,7 @@ Switch To New Tab
 Click On Searched Operator
     [Arguments]   ${operator}
      Search Installed Operator          ${operator}
-     Sleep   1
+     Wait Until Element is Visible     xpath=//a[@data-test-operator-row="${operator}"]    timeout=10
```

Comment:
would it be possible to use `Is Operator Installed` instead of line 83? they do the same thing

Reply:
I wouldn't use that...if we do that i have to add If condition for the code below since is operator install return ststaus..idea here is if it' doesn't list after 100 min it will fail as

Reply:
hmm I see..yeah it would be more a semantic issue having `Is Operator Installed` here..Okay :)